test/e2e/framework/helper/validate.go

```diff
@@ -33,30 +33,30 @@ type ValidationFunc func(certificate *cmapi.Certificate, secret *v1.Secret) erro
 func (h *Helper) DefaultValidationSet() []ValidationFunc {
 	return []ValidationFunc{
 		validations.Expect2Or3KeysInSecret,
-		validations.ExpectValidAnnotations,
-		validations.ExpectValidPrivateKeyData,
-		validations.ExpectValidCertificate,
 		validations.ExpectCertificateDNSNamesToMatch,
+		validations.ExpectCertificateOrganizationToMatch,
```

Comment:
The organization check was missing from this default list of validations, so I added it and sorted the list at the same time.